From 3b61453ba48f40d73aa1ad2c9e5a82a389920d0d Mon Sep 17 00:00:00 2001
From: Christian Tismer <tismer@stackless.com>
Date: Thu, 24 Nov 2022 10:24:58 +0100
Subject: [PATCH 2/2] Fix a cmake-only build, amended

pyminver was computed by cmake and evaluated by parser.py .
A recent refactoring extracted a computed string in a
wrong way.

Sorry, a tuple was needed as default.

Change-Id: Ib837af50f2cd525411a3a641e3ccc7fc99cf866a
Pick-to: 6.4
Fixes: PYSIDE-2127
Fixes: PYSIDE-2128
---
 .../files.dir/shibokensupport/signature/parser.py               | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sources/shiboken6/shibokenmodule/files.dir/shibokensupport/signature/parser.py b/sources/shiboken6/shibokenmodule/files.dir/shibokensupport/signature/parser.py
index 66d79ad..5d86b93 100644
--- a/sources/shiboken6/shibokenmodule/files.dir/shibokensupport/signature/parser.py
+++ b/sources/shiboken6/shibokenmodule/files.dir/shibokensupport/signature/parser.py
@@ -69,7 +69,7 @@ def _get_flag_enum_option():
         flag = getattr(sys, sysname)
         if not isinstance(flag, int):
             flag = True
-    p = f"\n    *** Python is at version {'.'.join(map(str, pyminver or 0))} now."
+    p = f"\n    *** Python is at version {'.'.join(map(str, pyminver or (0,)))} now."
     # PYSIDE-1797: Emit a warning when we may remove pep384_issue33738.cpp
     if pyminver and pyminver >= (3, 8):
         warnings.warn(f"{p} The file pep384_issue33738.cpp should be removed ASAP! ***")
-- 
2.38.1

